source/whats-new.txt

```diff
@@ -0,0 +1,318 @@
+.. _c-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade
+
+Learn what's new in:
+
+* :ref:`Version 1.27.6 <c-version-1.27.6>`
+* :ref:`Version 1.27.5 <c-version-1.27.5>`
+* :ref:`Version 1.27.4 <c-version-1.27.4>`
+* :ref:`Version 1.27.3 <c-version-1.27.3>`
+* :ref:`Version 1.27.2 <c-version-1.27.2>`
+* :ref:`Version 1.27.1 <c-version-1.27.1>`
+* :ref:`Version 1.27.0 <c-version-1.27.0>`
+* :ref:`Version 1.26.2 <c-version-1.26.2>`
+* :ref:`Version 1.26.1 <c-version-1.26.1>`
+* :ref:`Version 1.26.0 <c-version-1.26.0>`
+
+.. _c-upcoming-breaking-changes:
+
+.. _c-version-1.27.6:
+
+What's New in 1.27.6
+--------------------
+
+The 1.27.6 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.6 release of ``libbson`` includes the following changes:
+
```

Comment:
S: since you use a combination of "Changes," "Fixes," "New Features," and "Improvements" subheaders, I'd make sure each bullet point without a subheader is categorized as one of those four things (although I'm not sure "Changes" is necessary). In this case, you could add the "Fixes" subheader:
```suggestion
Fixes
`````
```

Reply:
Applies to all